PC02_Grafitti_Andrade_Hrusovsky.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+'''
+Turtle starter code
+ATLS 1300
+Author: Dr. Z
+Author: Erika Andrade
+May 29, 2020
+'''
+
+from turtle import * #import the library of commands that you'd like to use
+
+colormode(255)
+
+# Create a panel to draw on. 
+panel = Screen()
+w = 750 # width of panel
+h = 750 # height of panel
+panel.setup(width=w, height=h) #600 x 600 is a decent size to work on. 
+#You can experiment by making it the size of your screen or super tiny!
+
+# Create a colorful background and add Bezos image to it
+image = "Bezos.gif"
+panel.bgcolor("lightsteelblue1")
+panel.bgpic(image)
+
```

Comment:
try making multiple turtles to help organize your code and play at the same time
```suggestion
eyes = Turtle( ) #turtle draws red dots over bezos eyes
speachbubble = Turtle( ) #turtle creates a speachbubble 
RIGHT = Turtle( ) #turtle draws rectangle to right
LEFT = Turtle ( ) #turtle draws rectangle to right
speach = Turtle( ) #turtle writes inside of speachbubble
illuminati = Turtle( ) #turtle draws triangle on forehead
```